source/routing/tests/routing_api_test.cpp

```diff
@@ -3458,12 +3454,11 @@ BOOST_AUTO_TEST_CASE(journeys_with_time_frame_duration) {
     //-----------------------------------
     // Case 6 :
     // clockwise = true
-    // min_nb_journeys = 2
+    // min_nb_journeys = none
     // timeframe_duration = 40*60 (40min)
     // timeframe_max_bound = 1H
     //
     // the reponse should contain at least 4 journeys
-    min_nb_journeys = 2;
```

Comment:
Doesn't this change the test?

Reply:
Nope,we send `boost::none` to `make_response`in place of `min_nb_journeys`